src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -50,6 +50,8 @@ public CompilerOptions getCompilerOptions() {
     options.setClosurePass(true);
     options.setCheckGlobalNamesLevel(CheckLevel.ERROR);
     options.setCheckGlobalThisLevel(CheckLevel.ERROR);
+    options.setLanguage(CompilerOptions.LanguageMode.ECMASCRIPT6);
```

Comment:
Not sure if clutz can handle any ES6 module usage well. But this will enable other ES6 features that need .d.ts declarations, which I think is your usecase.